### PR TITLE
Drop Python 3.7, add Python 3.13, use latest meta/config templates

### PR DESCRIPTION
**Pull request by dataflake**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -48,14 +50,14 @@
...
+# source_encoding = 'utf-8-sig'
 
 # The master toctree document.
 master_doc = 'index'
 
 # General information about the project.
-project = u'transaction'
-copyright = u'2012, Zope Foundation Contributors'
+project = 'transaction'
+copyright = '2012, Zope Foundation Contributors'
```
Comment (icemac) [inline docs/conf.py:60]:
```suggestion
copyright = '2012-2024, Zope Foundation Contributors'
```

**Review by icemac — APPROVED** [review]
Let me have a look, why coverage has dropped below the threshold.

**Comment by icemac** [comment]
It seems to be because of `_compat.py` where not all code seems to be needed: https://coveralls.io/builds/69841069/source?filename=src%2Ftransaction%2F_compat.py
Maybe we can even get rid of the whole module.

**Comment by dataflake** [comment]
> It seems to be because of `_compat.py` where not all code seems to be needed: https://coveralls.io/builds/69841069/source?filename=src%2Ftransaction%2F_compat.py Maybe we can even get rid of the whole module.

That's now done. But the fact that coveralls complains about `-0.01%` is extremely annoying. Can you change that so it never complains unless there's at least a 0.5% change?

**Comment by icemac** [comment]
I changed the coveralls config and re-ran the tests.

BTW changing coveralls config is not depending on me. Using GitHub sign-in each contributor of ZF can change the settings there.

**Review by icemac — APPROVED** [review]

**Comment by dataflake** [comment]
Thank you!

**Pull request merged**